src/util/troll.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include "gta/enums.hpp"
+#include "services/players/player_service.hpp"
+#include "script_global.hpp"
+#include "pointers.hpp"
+
+namespace big::troll
+{
+	inline void set_bounty_on_player(player_ptr target, int value)
```

Comment:
Would be nice to have it as a command too.

Reply:
very easy to do lmao

Reply:
the commands system is 10/10 and its pretty easy to convert stuff into commands

Reply:
@Loko415 you still have got a problem with posting stuff into multiple comments for something that could've been edited in.